### ISSUE TITLE
[Bug]: Using Doughnut and Line chart is causing my component fail to render.

### ISSUE THREAD

**Issue by zeeshanejaz786** [reactchartjs/react-chartjs-2#1017]
### Would you like to work on a fix?

- [X] Check this if you would like to implement a PR, we are more than happy to help you go through the process.

### Current and expected behavior

I installed the latest version of chart.js and react-chartjs-2 using this command according to the Official documentation.

    npm install --save react-chartjs-2 chart.js

I am using datasets and I think I am doing everything right but it seems I am not.

  ```
    import React, { useEffect } from 'react';
    import Sidebar from './Sidebar.js';
    import './dashboard.css';
    import { Typography } from '@material-ui/core';
    import { Link } from 'react-router-dom';
    import { Doughnut, Line } from 'react-chartjs-2';
    import { useSelector, useDispatch } from 'react-redux';
    import { getAdminProduct } from '../../actions/productAction';
    import { getAllOrders } from '../../actions/orderAction.js';
    import { getAllUsers } from '../../actions/userAction.js';
    import MetaData from '../layout/MetaData';
```
    
    const Dashboard = () => {
      const dispatch = useDispatch();
    
      const { products } = useSelector((state) => state.products);
    
      const { orders } = useSelector((state) => state.allOrders);
    
      const { users } = useSelector((state) => state.allUsers);
    
      let outOfStock = 0;
    
      products &&
        products.forEach((item) => {
          if (item.Stock === 0) {
            outOfStock += 1;
          }
        });
    
      useEffect(() => {
        dispatch(getAdminProduct());
        dispatch(getAllOrders());
        dispatch(getAllUsers());
      }, [dispatch]);
    
      let totalAmount = 0;
      orders &&
        orders.forEach((item) => {
          totalAmount += item.totalPrice;
        });
    
      const lineState = {
        labels: ['Initial Amount', 'Amount Earned'],
        datasets: [
          {
            label: 'TOTAL AMOUNT',
            backgroundColor: ['tomato'],
            hoverBackgroundColor: ['rgb(197, 72, 49)'],
            data: [0, totalAmount],
          },
        ],
      };
    
      const doughnutState = {
        labels: ['Out of Stock', 'InStock'],
        datasets: [
          {
            backgroundColor: ['#00A6B4', '#6800B4'],
            hoverBackgroundColor: ['#4B5000', '#35014F'],
            data: [outOfStock, products.length - outOfStock],
          },
        ],
      };
    
      return (
        <div className="dashboard">
          <MetaData title="Dashboard - Admin Panel" />
          <Sidebar />
    
          <div className="dashboardContainer">
            <Typography component="h1">Dashboard</Typography>
    
            <div className="dashboardSummary">
              <div>
                <p>
                  Total Amount <br /> ${totalAmount}
                </p>
              </div>
              <div className="dashboardSummaryBox2">
                <Link to="/admin/products">
                  <p>Product</p>
                  <p>{products && products.length}</p>
                </Link>
                <Link to="/admin/orders">
                  <p>Orders</p>
                  <p>{orders && orders.length}</p>
                </Link>
                <Link to="/admin/users">
                  <p>Users</p>
                  <p>{users && users.length}</p>
                </Link>
              </div>
            </div>
    
            <div className="lineChart">
              <Line data={lineState} />
            </div>
    
            <div className="doughnutChart">
              <Doughnut data={doughnutState} />
            </div>
          </div>
        </div>
      );
    };
    
    export default Dashboard;

Here is the component I am using it in. 

if I remove the use of Doughnut and Line in the return statement the component works just fine.

### Reproduction

This component above is pretey much all i am using react-chartjs-2

### chart.js version

3.7.1

### react-chartjs-2 version

4.0.1

### Possible solution

I have no idea what is wrong here

**Comment by dangreen** [comment]
@zeeshanejaz786 Hi. Would you please describe what error/problem you got?

**Comment by samarsatyal** [comment]
I am having similar issue when using 'Line' in my component. I have the following code.. however, right after I add the line <Line data = {data /> inside my return ().. the entire component stops rendering.. the line/graph is never rendered for me. Any suggestions?

import { Line } from "react-chartjs-2";

const LineChart = () => {

const coinTimestamp = [33, 44, 55, 66, 77, 88, 99, 100];

const data = {
    labels: coinTimestamp,
    datasets: [
      {
        label: "Price In USD",
        data: [11, 22, 35, 45, 62, 55, 77],
        fill: false,
        borderColor: "rgb(75, 192, 192)",
        tension: 0.1,
      },
    ],
  };

return (
<>
<Line data = {data} />
</>
)};

**Comment by zeeshanejaz786** [comment]
> @zeeshanejaz786 Hi. Would you please describe what error/problem you got?



> @zeeshanejaz786 Hi. Would you please describe what error/problem you got?

My terminal shows no error.
it says compiled successfully because I have imported the right thing and I wrote the syntax correctly.

however, the console of the browser shows some errors. 
Here are the screenshots of the errors.

1. Uncaught Error: "category" is not a registered scale      [chart.ESM.js:4892 ]

![image](https://user-images.githubusercontent.com/95121238/154480976-2f23fc93-8c20-4ee5-ab0c-0fd0caaa95c5.png)


2. Uncaught Error: "arc" is not a registered element. 

![image](https://user-images.githubusercontent.com/95121238/154481092-308ec677-2a3e-42c2-b02a-81975863b612.png)

3. index.js:1 The above error occurred in the <ForwardRef(ChartComponent)> component:

![image](https://user-images.githubusercontent.com/95121238/154481201-63192835-e5c3-4ce1-af14-4cb2fe4d6ac4.png)


here is my terminal.
if I become the Doughnut and Line from the return statement it works fine.
else it compiles successfully but stops rendering the component.

![image](https://user-images.githubusercontent.com/95121238/154481610-37f029d2-6cf6-4ddb-afed-39f1e7bf7a1e.png)
